Add UEFI support (#2416)
cobbler/cobbler: docs/limitations.rst

Before templates are passed to Jinja or Cheetah there is a pre-processing of templates happening. During pre-processing
Cobbler replaces variables like ``@@my_key@@`` in the template. Those keys are currently limited by the regex of ``\S``,
which translates to ``[^ \t\n\r\f\v]``.

> Restarting the demon

Reviewer: ```suggestion
Restarting the daemon
```

====================

> Once you have a Cobbler Distro imported or manually added you have to make sure the source for the Kernel & Initrd is
> available all the time. Thus I highly recommend you to add the ISOs to your ``/etc/fstab`` to make them persistent
> across reboots. If you forget to remount then the Cobbler demon won't start!

Reviewer: ```suggestion
Once you have a Cobbler distro imported or manually added you have to make sure the source for the Kernel & initrd is
available all the time. Thus I highly recommend you to add the ISOs to your ``/etc/fstab`` to make them persistent
across reboots. If you forget to remount them the Cobbler daemon won't start!
```

> Kernel Options

Reviewer: ```suggestion
Kernel options
```
